src/client/components/app-header/app-header.vue

```diff
@@ -64,22 +67,25 @@ export default {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  position: sticky;
-  top: 0;
-  left: 0;
-  z-index: 10;
-  padding: 0 var(--spacing-default);
+  padding: 0 var(--spacing-half);
   width: 100%;
   height: var(--app-header-mobile-height);
   background-color: var(--background-color);
   box-shadow: 0 2px 15px 0 rgba(214,214,214,.5);
 }
 
+.app-header--sticky {
+  position: sticky;
+  top: 0;
+  left: 0;
+  z-index: 10;
```

Comment:
is 10 enough? Should we add a layer variables in there? Ask @celinekurpershoek  about this. It might not be necessary yet, but it is good to keep in mind as soon as we have modals (@jbmoelker will kill us but still...)